pages/strategy.md

```diff
@@ -0,0 +1,58 @@
+Exercism is a large complex project with thousands of contributors working across dozens of projects, all trying to achieve ambitious goals. It would quickly become overwhelming if we were all working towards making Exercism perfect, so to keep things manageable, we periodically determine the critical next step for Exercism to achieve its goals and then plan that into a strategy. Our current goal is to make Exercism scalable. We expect our next three goals to be to make Exercism financially sustainable, to make Exercism more accessible to beginners, and to intentionally grow Exercism for the first time.
+
+We present this as an Objective, which will aim to achieve with a Strategy, supported by a series of Tactics. This is commonly known as the OST framework.
+
+## Objective
+
+**Make Exercism scalable**
+
+At the time of writing, Exercism has 8,000 new submissions per week. Our mentors currently cannot keep up with the amount of submissions submitted. When they do keep up, the number of submissions almost immediately doubles, as students stick with a Language Track. We are growing by 300-400 new users per day. Many find Exercism too hard to use, but in the future, when that is fixed, we will expect to see the number of submissions double again. We have never actively marketed Exercism. All 300,000 members have come through word of mouth. When we start to market Exercism, we expect to see an increase in usage by two orders of magnitude. Our most important goal at this stage is being able to give our current students a timely mentoring experience, and be able to cope with further growth without overwhelming mentors.
+
+We aim to achieve this goal by the end of 2019.
+
+## Strategy
+
+**Acquire and retain more mentors by making mentoring rewarding and enjoyable**
+
+Currently, mentoring can be both fulfilling and frustrating. It can be very rewarding to help a student that wants to learn. On the flipside, it can be repetitive, students can be rude, there are few positive feedback loops, and it can feel too much like a job rather than a hobby. In order to retain our existing mentors, and to acquire new ones, we must make mentoring more enjoyable and more rewarding. We must make our mentors feel like that are part of something bigger them them, and that they are contributing to something valuable. A good parallel is wikipedia, where thousands of people contribute (from once a year, to dozens of times per day) because they feel like they are making a tangible difference to something that matters, and because the process itself is straight-forward but (strangely) rewarding.
```

Comment:
I think a semicolon separated list reads better here:
```markdown
On the flipside: it can be repetitive, students can be rude; there are few positive feedback loops; and it 
can feel too much like a job rather than a hobby
```

Reply:
>  We must make our mentors feel like that are part of something bigger them them,

 We must make our mentors feel like that are part of something bigger **than** them

Reply:
> A good parallel is wikipedia, where

A good parallel is *Wikipedia*, where

Reply:
>  straight-forward but (strangely) rewarding

 straight-forward **; and** (strangely) rewarding

Reply:
I'm not a huge fan of semi-colons here - I think commas do the job ok.